packages/shopping/src/sequence.ts

```diff
@@ -1,4 +1,4 @@
-// Copyright IBM Corp. 2018. All Rights Reserved.
+// Copyright IBM Corp. 2019. All Rights Reserved.
```

Comment:
Nitpick: `// Copyright IBM Corp. 2019. All Rights Reserved.` -> `// Copyright IBM Corp. 2018, 2019. All Rights Reserved.` (should be `{year created}, {year modified}`)